src/Performance/Memory/MemoryFormatter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Performance\Memory;
+
+/**
+ * @internal
+ */
+final class MemoryFormatter
+{
+    const BYTES_IN_MEGABYTE = 1024 * 1024;
+
+    public function toHumanReadableString(float $bytes): string
```

Comment:
I wonder if we can extract some kind of `Formatter` interface here.

Reply:
What about having only one formatter here? Even better having it in `PerformanceLoggerSubscriber`. Unless we have a need for it elsewhere, it doesn't really make sense to separate them and would reduce the number of classes needed

Reply:
> I wonder if we can extract some kind of Formatter interface here.

TBH I see no reason to extract the interface, because:

1. it is assumed to have only 1 implementation here
2. classes are final and internal, they are not supposed to be replaced by reimplementing the abstraction (interface) by users that use Infection
3. IMO memory and time formatter has nothing in common. It is a coincidence that they have method with the same name. `Formatter` does not suite here because it is too general thing, also memory and time formatters has different signature (`int`/`float` for parameter)
4. as stated before - it would be overkill here, keeping in mind that classes are internal and has 1 implementation

Reply:
> What about having only one formatter here? Even better having it in PerformanceLoggerSubscriber. 

Disagree. `PerformanceLoggerSubscriber` must not know how to format memory and time strings, this is violation of SRP from my POV. After moving this functionality to subscriber, it will be impossible to unit test it without making the methods `public`